css-pseudo-4/Overview.bs

```diff
@@ -675,7 +675,11 @@ Styling Highlights</h3>
     css/css-pseudo/active-selection-012.html
     css/css-pseudo/active-selection-031.html
     </wpt>
-    <li>'text-decoration' and its associated properties
+    <li>'text-decoration' and its associated properties, including
+    <ul>
+      <li>'text-underline-position'
+      <li>'text-underline-offset'
+    </ul>
```

Comment:
I might suggest doing this as 
```suggestion
    <li>'text-decoration' and its associated properties (including 'text-underline-position' and 'text-underline-offset')
```

so that it reads more as a clarification, and even less as an exhaustive list.

Reply:
done in 8c15aead880bc2a761dd3629cfe0e3480dd21c32